components/menu/index.css

```diff
@@ -10,170 +10,676 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+  --spectrum-menu-item-icon-height: var(--spectrum-workflow-icon-size-300);
+  --spectrum-menu-item-icon-width: var(--spectrum-workflow-icon-size-300);
+  --spectrum-menu-item-label-font-size: var(--spectrum-font-size-300);
+  --spectrum-menu-item-label-text-to-visual: var(--spectrum-text-to-visual-300);
+  
+  --spectrum-menu-item-label-inline-edge-to-content: var(--spectrum-component-edge-to-text-300);
+  --spectrum-menu-item-top-edge-to-text: var(--spectrum-component-top-to-text-300);
+  --spectrum-menu-item-bottom-edge-to-text: var(--spectrum-component-bottom-to-text-300);
+
+  --spectrum-menu-item-text-to-control: var(--spectrum-text-to-control-300);
+
+  --spectrum-menu-item-description-font-size: var(--spectrum-font-size-200);
+
+
+  --spectrum-menu-section-header-font-size: var(--spectrum-font-size-300);
+  --spectrum-menu-section-header-min-width: var(--spectrum-component-height-300);
+
+  /* TODO: eventual token name: */
+  /* --spectrum-menu-item-selectable-edge-to-text-not-selected: var(--spectrum-menu-item-edge-to-text-not-selected-extra-large); */
+  /* TODO: remove below once above is defined */
+  --spectrum-menu-item-selectable-edge-to-text-not-selected: 45px;
+
+  --spectrum-menu-item-checkmark-height: 14px;
+  --spectrum-menu-item-checkmark-width: 14px;
+}
+
+/* TODO: temporary & can be removed once the tokens are defined in the core tokens package */
+.spectrum--large {
+  .spectrum-Menu--sizeS {
+    --spectrum-menu-item-selectable-edge-to-text-not-selected: 34px;
+  }
+  .spectrum-Menu--sizeM {
+    --spectrum-menu-item-selectable-edge-to-text-not-selected: 42px;
+  }
+  .spectrum-Menu--sizeL {
+    --spectrum-menu-item-selectable-edge-to-text-not-selected: 47px;
+  }
+  .spectrum-Menu--sizeXL {
+    --spectrum-menu-item-selectable-edge-to-text-not-selected: 54px;
+  }
+}
+
+.spectrum--light {
+  --spectrum-menu-item-background-color-default: rgba(0, 0, 0, 0); /* --spectrum-gray-900 */
+  --spectrum-menu-item-background-color-hover: var(--spectrum-transparent-black-200);
+  --spectrum-menu-item-background-color-down: var(--spectrum-transparent-black-200);
+  --spectrum-menu-item-background-color-key-focus: var(--spectrum-transparent-black-200);
+}
+
+.spectrum--dark,
+.spectrum--darkest {
+  --spectrum-menu-item-background-color-default: rgba(255, 255, 255, 0); /* --spectrum-gray-900 */
+  --spectrum-menu-item-background-color-hover: var(--spectrum-transparent-white-200);
+  --spectrum-menu-item-background-color-down: var(--spectrum-transparent-white-200);
+  --spectrum-menu-item-background-color-key-focus: var(--spectrum-transparent-white-200);
 }
 
 .spectrum-Menu {
-  @remapvars {
-    find: /--spectrum-listitem-m(.*)/;
-    filter: color;
-    replace: --spectrum-listitem$1;
+  --spectrum-menu-item-label-line-height: var(--spectrum-line-height-100);
+  --spectrum-menu-item-label-cjk-line-height: var(--spectrum-cjk-line-height-100);
+  --spectrum-menu-item-label-to-description-spacing: 1px;
```

Comment:
Should this either have a token or go in the Dropbox Paper doc?

Reply:
Oh, you're right! Thank you.